deploy/eck-stack/charts/eck-enterprise-search/examples/with-custom-configuration.yaml

```diff
@@ -0,0 +1,19 @@
+config:
+  # define the exposed URL at which users will reach Enterprise Search
+  ent_search.external_url: https://my-custom-domain:3002
+  # define the exposed URL at which users will reach Kibana
+  kibana.host: https://kibana.my-custom-domain:5601
```

Comment:
Maybe worth having these two commented out in the main `values.yaml` given how important they are. With a link to https://www.elastic.co/guide/en/cloud-on-k8s/current/k8s-enterprise-search-configuration.html maybe.